pkg/controller/statefulset/stateful_set.go

```diff
@@ -114,28 +110,33 @@ func NewStatefulSetController(podInformer cache.SharedIndexInformer, kubeClient
 			},
 			DeleteFunc: ssc.enqueueStatefulSet,
 		},
+		statefulSetResyncPeriod,
```

Comment:
Not necessary I think but an issue to extend the resync period and also add the PVC informer would be nice.